iotedgedev/modules.py

```diff
@@ -15,6 +14,53 @@ def __init__(self, envvars, utility, output, dock):
         self.dock = dock
         self.dock.init_registry()
 
+    def create(self, name, template):
+        self.output.header("CREATING MODULE")
+
+        repo = "{0}/{1}".format(self.envvars.CONTAINER_REGISTRY_SERVER, name.lower())
+        cwd = self.envvars.MODULES_PATH
+        if template == "csharp":
+            cmd = "dotnet new -i Microsoft.Azure.IoT.Edge.Module"
+            self.output.header(cmd)
+            self.utility.exe_proc(cmd.split())
+            cmd = "dotnet new aziotedgemodule -n {0} -r {1}".format(name, repo)
+            self.output.header(cmd)
+            self.utility.exe_proc(cmd.split(), cwd=cwd)
+        elif template == "nodejs":
+            pass
```

Comment:
Consider using the node_solution template that is in the test folder.  Let's figure out a way to support node with a template.

Reply:
We have created a [Yeoman template](https://www.npmjs.com/package/generator-azure-iot-edge-module) for Node.js module. We would like to announce and document it but in the last minute were informed of the potential breaking changes introduced by edgelet. After it is fully tested on the new runtime, we can integrate it here.